Movie/MoviesMVP/MovieListViewController/Cell/AboutMovieTableViewCell.swift

```diff
@@ -60,12 +60,19 @@ final class AboutMovieTableViewCell: UITableViewCell {
 
     // MARK: - Public Methods
 
-    func configure(movieName: String, imageName: String, rating: Double) {
-        movieNameLabel.text = movieName
-        ratingLabel.text = String(rating)
-        coverMovieImageView.loadImage(
-            urlName: Constants.imagePath + imageName
-        )
+    func configure(movie: Movie, networkService: NetworkServiceProtocol?) {
+        movieNameLabel.text = movie.title
+        ratingLabel.text = String(movie.voteAverage)
+        guard let coverMovieImageURL = URL(string: Constants.imagePath + movie.posterPath)
+        else { return }
+        networkService?.loadDataFrom(url: coverMovieImageURL, completion: { [weak self] result in
```

Comment:
```suggestion
        networkService?.loadDataFrom(url: coverMovieImageURL, completion: { [weak self] result in
```
Это делаем в отдельном методе